terraform/cloudfront.tf

```diff
@@ -28,8 +28,8 @@ resource "aws_cloudfront_distribution" "s3_distribution" {
 
     viewer_protocol_policy = "redirect-to-https"
     min_ttl                = 0
-    default_ttl            = 3600  // 1 hour
-    max_ttl                = 86400 // 1 day
+    default_ttl            = 0
+    max_ttl                = 0
```

Comment:
Probably just give me 5 minutes to get over this but the preferred option would be to invalidate the cache on the [CloudFront distribution through the AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/cloudfront/create-invalidation.html) but I am also aware of the effort that will probably take to do that.

Reply:
Curious - why would that be the preferred option?  Feels hacky having to invalidate a cache we don't require in the first place?

Reply:
From a bit of reading online (not at all exhaustive) I'm getting the sense it's a common pattern to use CloudFront for non-caching stuff like DDoS protection and HTTPS (as we are), even if you don't need the caching element?